projects/bp-gallery/src/graphql/operation.graphql

```diff
@@ -166,12 +166,51 @@ query getCollectionInfo($collectionName: String) {
   }
 }
 
+query getCollectionInfoById($collectionId: ID!) {
+  collection(id: $collectionId) {
+    data {
+      id
+      attributes {
+        name
+        description
+        child_collections(sort: "name:asc") {
+          data {
+            id
+            attributes {
+              name
+              pictures(pagination: { limit: 1 }) {
+                data {
+                  id
+                }
+              }
+              child_collections(pagination: { limit: 1 }) {
```

Comment:
This is needed so that we know if there even is a child collection (hence the `limit: 1`). This is so that we can quickly check whether we could, for example, delete the collection. This operation should only be possible if the collection does not have any children, and checking it here makes the code down the road more readable.